hazelcast/src/main/java/com/hazelcast/internal/memory/MemoryAccessor.java

```diff
@@ -61,6 +62,12 @@
      */
     MemoryAccessor AMEM = MemoryAccessorProvider.getMemoryAccessor(STANDARD);
 
+
+    /**
+     * The instance of {@link MemoryAccessor} which  handles data in specified heap-byteBuffer (byte[] buffer).
+     */
+    MemoryAccessor HEAP_BYTE_BUFFER_MEM = MemoryAccessorProvider.getMemoryAccessor(HEAP_BYTE_BUFFER);
```

Comment:
This file has a huge diff, but this seems to be the only semantic change. Could you make the diff more focused on the actual change? I don't see the value in any of the other changes.